app/main.py

```diff
@@ -56,6 +72,15 @@ def upload():
         if filename != "default.png.csv":
             file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
         generate_badges()
+
+        # Remove the uploaded files after job in done
+        if filename != "default.png.csv":
+            os.unlink(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+        try:
+            os.unlink(os.path.join(app.config['UPLOAD_FOLDER'], imgname))
+        except Exception:
+            pass
```

Comment:
```
>>> import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
Readability counts.
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```
The zen of Python.

> Errors should never pass silently.

This is very important to me.
You can print the last error always, using
```
import traceback
traceback.print_exc()
```

Reply:
I merge it and create an issue, to keep the development flow going.

Reply:
@niccokunzmann Are you looking for logging the error message at the error log, or an error message displayed to the user?

Reply:
An error message in the logs would be good for debugging. Maybe also to the user, I do not know if that would be important to the user.